crates/oxc_semantic/src/post_transform_checker.rs

```diff
@@ -421,11 +421,11 @@ impl<'s> PostTransformChecker<'s> {
             }
 
             // Check children match
-            let child_ids = self
-                .get_pair(scope_ids, |data, scope_id| data.scopes.get_child_ids(scope_id).to_vec());
-            if self.remap_scope_ids_sets(&child_ids).is_mismatch() {
-                self.errors.push_mismatch("Scope children mismatch", scope_ids, child_ids);
-            }
+            // let child_ids = self
+            // .get_pair(scope_ids, |data, scope_id| data.scopes.get_child_ids(scope_id).to_vec());
+            // if self.remap_scope_ids_sets(&child_ids).is_mismatch() {
+            // self.errors.push_mismatch("Scope children mismatch", scope_ids, child_ids);
+            // }
```

Comment:
@overlookmotel how do you want to change this?

Reply:
We can just delete this chunk entirely. No child IDs = no child IDs to check. One less thing to get wrong!